tests/core/data_model/test_dataobject.py

```diff
@@ -517,6 +517,32 @@ class Foo(DataObjectBase):
     assert foo2.foo_bool2
 
 
+@pytest.mark.skip(reason="wip")
```

Comment:
worth adding some more context on what the WIP is? Even I can't remember now 🤔

Reply:
yes! I updated the reason to be that we still cannot use static BoolSequence in core (issue https://github.com/caikit/caikit/issues/409). But correct me if that's not the reason. The error seen if this test is run is:
```
...
except TypeError as e:
                # More likely than not, this is a duplicate symbol; the main case in which
                # this could occur is when you've compiled files with protoc, added them to your
                # descriptor pool, and ALSO added the defs in your py_to_proto schema, but the
                # lookup validation with fd_proto.name is skipped because the .proto file fed to
                # protoc had a different name!
>               raise TypeError(
                    f"Failed to add {fd_proto.name} to descriptor pool with error: [{e}]; Hint: if you previously used protoc to compile this definition, you must recompile it with the name {fd_proto.name} to avoid the conflict."
                )
                
                
E               TypeError: Failed to add foo.proto to descriptor pool with error: [Couldn't build proto file into descriptor pool!
E               Invalid proto descriptor for file "foo.proto":
E                 caikit_data_model.Foo.FooBoolSequence.values: "values" is already defined in "caikit_data_model.Foo.FooBoolSequence".
E                 caikit_data_model.Foo.FooBoolSequence: "FooBoolSequence" is already defined in "caikit_data_model.Foo".
E               ]; Hint: if you previously used protoc to compile this definition, you must recompile it with the name foo.proto to avoid the conflict.

venv/lib/python3.9/site-packages/py_to_proto/utils.py:67: TypeError
```

Reply:
Yeah that sounds good (technically it's more of we `don't` use static BoolSequence in core instead of `can't`, at the moment)